### PR TITLE
[css-multicol] Add tests related to column-gap animations

### PR DESCRIPTION
**Pull request by mrego**
Maybe @rachelandrew and/or @frivoal can take a look to these tests. Thanks!

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22217)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/322860598)

File: css/css-multicol/multicol-gap-animation-001.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Multi-column Layout Test: column-gap test animation</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="https://www.w3.org/TR/css-multicol-1/#column-gap">
+<link rel="help" href="https://www.w3.org/TR/web-animations-1/#dom-animatable-animate">
+<meta name="assert" content="Test checks that colunm-gap is an animatable property, and that the computed values during the animation are the expected ones.">
```
Comment (mstensho) [inline css/css-multicol/multicol-gap-animation-001.html]: *column-gap

File: css/css-multicol/multicol-gap-animation-001.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          duration: 10,
+          fill: "forwards"
+        };
+
+        var player = multicol.animate(keyframes, options);
+        player.pause();
+        player.currentTime = 0;
+        assert_equals(getComputedStyle(multicol).columnGap, "100px");
+        player.currentTime = 5;
+        assert_equals(getComputedStyle(multicol).columnGap, "150px");
```
Comment (mstensho) [inline css/css-multicol/multicol-gap-animation-001.html]: With the initial timing function 'ease', can you really expect it to be at 150px here?

File: css/css-multicol/multicol-gap-animation-002.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  @keyframes column-gap-normal-to-100 {
+    from { column-gap: normal; }
+    to { column-gap: 100px; }
+  }
+
+  #multicol {
+    animation: column-gap-normal-to-100 2s -1s paused;
```
Comment (mstensho) [inline css/css-multicol/multicol-gap-animation-002.html]: Could you use longhands instead, please? I don't know the animations spec too well, and have a hard time reading this.

File: css/css-multicol/multicol-gap-animation-003.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+<meta name="assert" content="Test checks that the default value for column-gap property, which is 'normal', is not interpolable.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  @keyframes column-gap-to-100 {
+    to { column-gap: 100px; }
+  }
+
+  #multicol {
+    animation: column-gap-to-100 2s -1s paused;
```
Comment (mstensho) [inline css/css-multicol/multicol-gap-animation-003.html]: ditto

**Review by mstensho — CHANGES_REQUESTED** [review]

File: css/css-multicol/multicol-gap-animation-001.html (hunk 1 above)
Comment (mrego) [reply css/css-multicol/multicol-gap-animation-001.html]: Done.

File: css/css-multicol/multicol-gap-animation-001.html (hunk 2 above)
Comment (mrego) [reply css/css-multicol/multicol-gap-animation-001.html]: I believe [the default one is "linear"](https://drafts.csswg.org/web-animations/#time-transformations), but I can explicitly write it.

File: css/css-multicol/multicol-gap-animation-002.html (hunk 3 above)
Comment (mrego) [reply css/css-multicol/multicol-gap-animation-002.html]: Done.

File: css/css-multicol/multicol-gap-animation-003.html (hunk 4 above)
Comment (mrego) [reply css/css-multicol/multicol-gap-animation-003.html]: Done.

**Comment by mrego** [comment]
@mstensho thanks for the review, I've just uploaded a new version fixing the issues you mentioned.

File: css/css-multicol/multicol-gap-animation-001.html (hunk 2 above)
Comment (mstensho) [reply css/css-multicol/multicol-gap-animation-001.html]: Oh, I was reading here: https://www.w3.org/TR/css-animations-1/#animation-timing-function

How confusing.

**Review by mstensho — APPROVED** [review]

**Pull request merged**